src/components/SideNav/SideNavItem.js

```diff
@@ -15,25 +20,20 @@ export default class SideNavItem extends React.Component {
     });
   };
 
-  renderSubNavItems = (subItems, location) =>
-    Object.keys(subItems).map(item => {
-      const pathname = normalizeLocation(location);
-      let isItemActive = false;
-
-      // Support nested /components/:component/:tab
-      // Support nested /guidelines/:guideline/:tab
-      if (pathname.length === 3) {
-        // Match :component, :guideline
-        isItemActive = locationContainsPathAtIndex(location, item, -2);
-      }
-
-      // Support /components/overview
-      if (pathname.length === 2) {
-        isItemActive = locationContainsPathAtIndex(location, item, -1);
-      }
+  renderSubNavItems = (subItems, location) => {
+    return Object.keys(subItems).map(item => {
+      // Whether or not a side nav "item" is activated. Only navItems that are
+      // routes will ever be active.
+      let isNavItemActive = locationContainsPathAtIndex(
```

Comment:
As a style nit, we tend to reserve `let` only for mutable bindings and values. As a result, the tendency is to use `const` by default unless the value changes 👍

Reply:
Roger that!